spark-jobs/src/test/scala/filodb/downsampler/DownsamplerMainSpec.scala

```diff
@@ -400,137 +447,6 @@ class DownsamplerMainSpec extends AnyFunSpec with Matchers with BeforeAndAfterAl
     }
   }
 
-  it ("should correctly generate export address from labels") {
-    val testConf = ConfigFactory.parseString(
-        """
-          |    filodb.downsampler.data-export {
-          |      enabled = true
-          |      key-labels = ["l1", "l2"]
-          |      destination-format = "address/foo/bar/$0/$1"
-          |    }
-          |""".stripMargin
-      ).withFallback(conf)
-
-    val dsSettings = new DownsamplerSettings(testConf.withFallback(conf))
-    val batchExporter = new BatchExporter(dsSettings, dummyUserTimeStart, dummyUserTimeStop)
-
-    batchExporter.makeExportAddress(Seq("l1a", "l2a")) shouldEqual "address/foo/bar/l1a/l2a"
-  }
-
-  it ("should give correct export-row index of column name") {
-    val testConf = ConfigFactory.parseString(
-      """
-        |    filodb.downsampler.data-export {
-        |      enabled = true
-        |      key-labels = ["l1", "l2"]
-        |      path-spec: [
-        |        year, "<<YYYY>>",
-        |        month, "<<M>>",
-        |        day, "<<d>>",
-        |        _metric_, "{{__name__}}"
-        |      ]
-        |    }
-        |""".stripMargin
-    ).withFallback(conf)
-
-    val dsSettings = new DownsamplerSettings(testConf.withFallback(conf))
-    val batchExporter = new BatchExporter(dsSettings, dummyUserTimeStart, dummyUserTimeStop)
-    batchExporter.getRowIndex("year").get shouldEqual 3
-    batchExporter.getRowIndex("month").get shouldEqual 4
-    batchExporter.getRowIndex("day").get shouldEqual 5
-    batchExporter.getRowIndex("_metric_").get shouldEqual 6
-
-  }
```

Comment:
This is a good test to keep just to be sure the new implementation works.

Reply:
Added test case for getColumnIndex and few more.